docs/integrations/github_actions/aws_oidc.md

```diff
@@ -0,0 +1,170 @@
+
+In order to checkout to the GitHub repository and to save the benchmark results to the repository, your job or workflow run also requires a permissions setting with `contents: write`.
+
+```yaml
+permissions:
+  id-token: write
+  contents: write
+```
+
+### Steps
+
+A workflow comprises one or more jobs that run in parallel, each with one or more steps that run in order. Our [example](https://github.com/turbot/steampipe-samples/blob/d1658920c41da9ff5a2b98ab981ad330f2ee34a8/all/github-actions-oidc/aws/steampipe-sample-aws-workflow.yml#L15) defines a single job with a series of steps that authenticate to AWS, install Steampipe, run a compliance benchmark and save the results to the repository.
+
+First, create a step that configures the credentials Steampipe will use to access AWS.
+
+```yaml
+- name: "Configure AWS credentials"
+  id: config-aws-auth
+  uses: aws-actions/configure-aws-credentials@v1-node16
+  with:
+    role-to-assume: ${{ secrets.OIDC_AWS_ROLE_TO_ASSUME }}
+    role-session-name: "steampipe-demo"
+    role-duration-seconds: 900
+    aws-region: "us-east-1"
+```
+
+Once the cloud provider successfully validates the claims presented in the OIDC JWT ID token, it then provides a short-lived access token that is available only for the duration of the job. The short-lived access token is exported as environment variables AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and AWS_SESSION_TOKEN.
+Steampipe will load these short-lived [credentials from Environment variables](https://hub.steampipe.io/plugins/turbot/aws#credentials-from-environment-variables) to run the benchmark.
+
+Next, you'll need to create a step that installs the Steampipe CLI and AWS plugin.
+
+```yaml
+- name: "Install Steampipe cli and plugin"
+  id: steampipe-installation
+  run: |
+
+    # Install Steampipe CLI
+    sudo /bin/sh -c "$(curl -fsSL https://raw.githubusercontent.com/turbot/steampipe/main/install.sh)"
+    # Check steampipe version
+    steampipe -v
+    # Install AWS plugin
+    steampipe plugin install aws
+```
+
+Before running the compliance benchmark, create a new folder on the branch specified in your GitHub repository to save the benchmark output. In our example, we will save the outputs to the folder `steampipe/benchmarks/aws`. The default environment variable [GITHUB_WORKSPACE]((https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables)) refers to the default working directory on the runner for steps, and the default location of your repository when using the [checkout](https://github.com/actions/checkout) action.
+
+Next, create a step that runs the [AWS Compliance](https://hub.steampipe.io/mods/turbot/aws_compliance) mod.
```

Comment:
I think it would be more accurate to say that it installs the AWS Compliance mod and runs the AWS CIS Benchmark